main/zoe/guide/contracts/public-auction.md

```diff
@@ -105,19 +105,22 @@ const bobOfferRules = harden({
     kind: 'onDemand',
   },
 );
+
 const bobPayments = [undefined, bobSimoleanPayment];
-const {
-  escrowReceipt,
-  payout,
-} = await zoe.escrow(bobPayoutRules, bobPayments);
 
-auction.bid(bobEscrowReceipt);
+const { seat: bobSeat, payout: bobPayoutP } = await zoe.redeem(
+  bobExclusiveInvite,
+  bobOfferRules,
+  bobPayments,
+);
+
+const bobOfferResult = await bobSeat.matchOffer();
 ```
 
-And let's say that Carol and Dave also decide to bid in the same way
+Since multiple parties may want to participate in the swap, let's say that Carol and Dave also decide to bid in the same way
```

Comment:
Let's call it an auction rather than a swap

Reply:
I've been wondering about that. Sometimes "swap" and "auction" seem interchangeable. Do you have distinct preferences as to when to use one over the other?

Reply:
Hmm, I've been using 'swap' for anything in which alice gets all of bob's stuff and vice versa. For this auction contract, there are multiple parties, and the winner is decided by the second highest price. If there's any mention of a swap referring to an auction that's probably a mistake on my part.